data/districts.json

```diff
@@ -7,7 +7,7 @@
   },
   {
-    "name": "Ilam",
+    "name": "Illam",
```

Comment:
Renaming won't be a good solution so we are implementing this #18, so can you please update the PR by removing this file?